### ISSUE TITLE
[BUG] Shieldbow does not work when cdragon isn't updated yet

### ISSUE THREAD

**Issue by DarkIntaqt** [TheDrone7/shieldbow#61]
**Describe the bug**
Not setting a version in the client config (so using latest) does completely break shieldbow if the new version is not already on cdragon. 

**Expected behavior**
It should downgrade to the prior patch or have a better error handling in place. 

**Error**
```sh
$ npm run test
```
output for champion.test.ts:
```
 FAIL  tests/champion.test.ts
  DRAGON: champions
    ✕ can fetch a champion by ID
    ✕ can fetch champion from cache (132 ms)
    ✕ can fetch champion from storage (120 ms)
    ✕ can fetch champion by name and key (cached) (336 ms)
    ✕ can fetch champion by name and key (storage) (319 ms)
    ✕ can fetch champion by name and key (forced) (315 ms)
    ✕ can fetch all champions (313 ms)
    ✕ assigns champion spells correctly (1 ms)
    ✕ contains champion passive (1 ms)
    ✕ contains champion classes
    ✕ can fetch champion assets
    ✕ can cache champions (6 ms)
    ✕ can fetch champion sprites (1 ms)
    ✕ can fetch champion rotations (1 ms)
    ✕ can fetch champion rotations from cache (1261 ms)
```

**Desktop (please complete the following information):**
 - Node.JS version [v16.15.0]
 - Library version [2.1.0]
- Typescript version [^4.9.5]

**Additional context**
https://github.com/TheDrone7/shieldbow/blob/bddd6edc2fe490aa5ce4167d3c665128726a6fe2/src/managers/ChampionManager.ts#L330
This line is most likely causing the trouble, as long as this line does not check for prior versions if the current isn't fully released yet 
https://github.com/TheDrone7/shieldbow/blob/bddd6edc2fe490aa5ce4167d3c665128726a6fe2/src/client.ts#L433

**Comment by TheDrone7** [comment]
Yep, I was already on fixing this, (as a follow up on the conversation in #lol-dev in the server)

**Comment by TheDrone7** [comment]
Addressed, will be shipped with v2.1.1
